docs/guide/doc-ops.rst

```diff
@@ -0,0 +1,71 @@
+DocOps
+======
+
+author: Jodie Putrino & the Write the Docs community
+
+What is DocOps, anyway?
+-----------------------
+
+Along with the advent of :doc:`/guide/docs-as-code`, the technical writing community 
+has seen the growth of a new specialization called "DocOps" (or "doc-ops").
+
+But what does DocOps actually mean, and who actually "does" DocOps?
+
+This subject came up for discussion during the WTD NA 2021 Unconference session. 
+This page represents some of the ideas presented during that session.
+
+Generally speaking, "DocOps" is like `DevOps`_. Instead of applying broadly to 
+software development, though, DocOps specifically applies to the creation, 
+management, and release of documentation.  
+
+Going from the DevOps definition provided by Atlassian linked in the previous 
+paragraph, then, we could loosely define DocOps as:
+
+"a set of practices that works to automate and integrate the process 
+of developing documentation across engineering, product, support, and, of 
+course, technical writing teams." 
+
+Just like DevOps, DocOps also emphasizes collaboration and continuous 
+improvement. Below, we've adapted the traditional DevOps lifecycle diagram to
+represent DocOps:
+
+.. image:: /_static/img/guide/docops-lifecycle.png
+
+.. _DevOps: https://www.atlassian.com/devops
+
+Who practices DocOps?
+---------------------
+
+To answer this question, it might be best to examine the responsibilities 
+that a DocOps practitioner or team might have. 
+
+* Selecting and providing support for tools used to produce documentation -- markup 
+  language, static site generator (SSG), hosting provider, etc.
+* Defining processes and established ways of working, including where docs fit
+  into agile or other development methodologies.
+* Defining documentation priorities and deliverables
+* Providing training to authors to enable contribution to the docs
+* Managing content     
+* Building and maintaining a theme for an SSG
```

Comment:
This is a great list. As a DocOps practitioner, I also model product divergence (versioning) of all kinds and map it to the documentation infrastructure. The main reason teams hire me is to implement version management. And as long as we're mentioning theming, we're also usually maintaining build scripts to coordinate it all and make docs building as simple as possible on the commandline.

Basically, I tell writing teams that in the end I'm going to be their command-line concierge.

Reply:
ooh, excellent suggestions!

```suggestion
* Building and maintaining a theme for an SSG
* Modeling and implementing product versioning strategies as they pertain to documentation
* Creating and maintaining build scripts 
* Keeping the barrier to entry for making docs low to make contributing easy and (dare we say it?) fun
```